jdat_notebooks/MRS_Mstar_analysis/JWST_Mstar_dataAnalysis_PointSourceDetectorBasedExtraction.ipynb

```diff
@@ -8,8 +8,7 @@
     }
    },
    "source": [
-    "<a id=\"top\"></a>\n",
```

Comment:
Is the removal of HTML tag intentional?

Reply:
Hmmm, I didn't know what that was. I removed it in all cases. What's that even there for?

Reply:
I think it is for "jump to top" HTML functionality but you have to ask the author to be sure.

Reply:
Ah, I am fine with overriding that. In the future, we can make that consistent in all notebooks.